client/screens/feed/Feed.js

```diff
@@ -174,18 +169,18 @@ const Feed = ({ feedType = 'public' }) => {
     trip: false,
   });
   const [searchQuery, setSearchQuery] = useState('');
-
+  const [Data,setData]=useState(data)
   const [refreshing, setRefreshing] = useState(false);
 
   const dispatch = useDispatch();
   const ownerId = useSelector((state) => state.auth.user?._id);
-  // const publicPacksData = useSelector((state) => state.feed.publicPacks);
-  // const userPacksData = useSelector(selectAllPacks);
```

Comment:
We don't want to use redux for these things

Reply:
yeah most of them is not being used now but for removing auth we have to carefully remove it